agent.py

```diff
@@ -40,6 +40,7 @@ def __init__(self, agentID, birthday, cell, configuration):
         self.sugar = configuration["sugar"]
         self.sugarMetabolism = configuration["sugarMetabolism"]
         self.tags = configuration["tags"]
+        self.tagging = configuration["tags"]
```

Comment:
This should be `configuration["tagging"]` or else `doTagging` will happen even if tagging is set to false.

Reply:
Done.